modules/system/console/OctoberEnv.php

```diff
@@ -394,4 +409,31 @@ private function dbConfig()
         ];
     }
 
+    /**
+     * Config (env) keys of which the value should not be included in orginal config files anymore.
+     *
+     * @return array
+     */
+    private function protectedConfig()
```

Comment:
This should either be named `getDefaultConfig()` or made available as a `defaultConfig` property on the class.

Reply:
I didn't mean `$defaultConfig = $this->protectedConfig`. I meant literally
```php
$defaultConfig = [
    'APP_KEY' => 'CHANGE_ME!!!!!!!!!!!!!!!!!!!!!!!',
];
```

Reply:
My bad. Do you mean like I did it now?